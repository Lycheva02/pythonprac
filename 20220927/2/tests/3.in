17, 4, 62, 798, 42 
