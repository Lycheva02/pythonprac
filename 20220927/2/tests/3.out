[798, 4, 62, 42, 17]
